streaming/src/main/scala/org/apache/gearpump/streaming/task/Subscription.scala

```diff
@@ -40,7 +41,7 @@ class Subscription(
     executorId: Int,
     taskId: TaskId,
     subscriber: Subscriber, sessionId: Int,
-    transport: ExpressTransport,
+    sender: TaskActor,
```

Comment:
I think here `sender` is just the current task actor, while literally sender means the upstream task actor